src/main/java/fi/metatavu/metamind/kuntaapi/KuntaApiConsts.java

```diff
@@ -0,0 +1,12 @@
+package fi.metatavu.metamind.kuntaapi;
+
+public class KuntaApiConsts {
+  
+  public static final String APIURL_SETTING_KEY = "kuntaapi-url";
+  public static final String USERNAME = "kuntaapi-username";
+  public static final String PASSWORD = "kuntaapi-password";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PASSWORD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)